GameData/RealismOverhaul/Engine_Configs/Aestus_Config.cfg

```diff
@@ -1,65 +1,178 @@
-	
+	%description = A hypergolic upper stage engine. The baseline Aestus engine is pressure fed and it used on the Ariane 5 G, GS and ES series, while the Aestus II (also known as RS-72) variant is pump-fed and develops a higher overall performance. Diameter: 1.32 m.
+
+	@MODULE[ModuleEngines*]
+	{
+		%minThrust = 30.0
+		%maxThrust = 30.0
+		%heatProduction = 36
+		@useEngineResponseTime = False
+		@useThrustCurve = False
+		%EngineType = LiquidFuel
+		%ullage = True
+		%pressureFed = True
+		%ignitions = 5
+
+		IGNITOR_RESOURCE,*{}
+
+		!thrustCurve,*{}
+	}
+
+	@MODULE[ModuleGimbal],*
+	{
+		@gimbalRange = 9.5
+		%useGimbalResponseSpeed = True
+		%gimbalResponseSpeed = 16
+	}
+
+	!MODULE[ModuleEngineConfigs],*{}
+
 	MODULE
 	{
 		name = ModuleEngineConfigs
-		origMass = 0.1665
+		type = ModuleEngines
 		configuration = Aestus
-		modded = false
+		origMass = 0.111
+
 		CONFIG
 		{
 			name = Aestus
-			maxThrust = 30
-			minThrust = 30
+			minThrust = 30.0
+			maxThrust = 30.0
+			heatProduction = 36
+			massMult = 1.0
+			ullage = True
+			pressureFed = True
+			ignitions = 5
+
+			IGNITOR_RESOURCE
+			{
+				name = ElectricCharge
+				amount = 0.25
+			}
+
 			PROPELLANT
 			{
 				name = MMH
-				ratio = 0.464446
+				ratio = 0.4684
 				DrawGauge = true
 			}
+
 			PROPELLANT
 			{
 				name = NTO
-				ratio = 0.535554
+				ratio = 0.5316
+				DrawGauge = False
 			}
+
 			atmosphereCurve
 			{
 				key = 0 324
 				key = 1 113
 			}
-			
+		}
+
+		CONFIG
+		{
+			name = Aestus-II
+			minThrust = 55.4
+			maxThrust = 55.4
+			heatProduction = 56
+			massMult = 1.2432
 			ullage = True
 			pressureFed = True
-			ignitions = 0 //unlimited
+			ignitions = 3
 
 			IGNITOR_RESOURCE
 			{
+				name = ElectricCharge
+				amount = 0.25
+			}
+
+			PROPELLANT
+			{
 				name = MMH
-				amount = 0.464446
+				ratio = 0.4684
+				DrawGauge = True
 			}
-			IGNITOR_RESOURCE
+
+			PROPELLANT
 			{
 				name = NTO
-				amount = 0.535554
+				ratio = 0.5316
+				DrawGauge = False
+			}
+
+			atmosphereCurve
+			{
+				key = 0 340
+				key = 1 150
 			}
 		}
 	}
-	!RESOURCE[ElectricCharge]
-	{
-	}
-	!MODULE[ModuleAlternator]
+
+	!MODULE[ModuleAlternator],*{}
+
+	!RESOURCE,*{}
+}
+
+//  ==================================================
+//  TestFlight compatibility.
+//  ==================================================
+
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[Aestus]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
+{
+	TESTFLIGHT
 	{
+		name = Aestus
+		ratedBurnTime = 1100
+		ignitionReliabilityStart = 0.968
+		ignitionReliabilityEnd = 0.98
+		cycleReliabilityStart = 0.97
+		cycleReliabilityEnd = 0.985
 	}
-	@MODULE[ModuleGimbal]
+}
+
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[Aestus-II]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
+{
+	TESTFLIGHT
 	{
-		@gimbalRange = 5
-		%useGimbalResponseSpeed = true
-		%gimbalResponseSpeed = 16
+		name = Aestus-II
+		ratedBurnTime = 600
```

Comment:
If TF works how i remember, then the engine is likely to fail after 600s with this. my source says design life of 2500s